setup.py

```diff
@@ -13,9 +13,9 @@
 
 
 INSTALL_REQUIRES = [
-    "transformers == 4.40.2",
+    "transformers == 4.41.1",
```

Comment:
could you test before the release (with perhaps test pypi) that `pip install optimum[neuronx]` installs the correct version of optimum and optimum-neuron? thx